tasks/managed/push-disk-images/push-disk-images.yaml

```diff
@@ -78,14 +84,16 @@ spec:
         fi
 
         echo "Creating InternalRequest to push disk images..."
-        internal-request -r "push-disk-images" \
+        internal-request --pipeline "push-disk-images" \
```

Comment:
This won't work. You need to move the internal pipeline to the pipelines dir. It is still in internal/pipelines. The internalrequest script only looks in one spot for these https://github.com/konflux-ci/release-service-utils/blob/main/utils/internal-request#L257